packages/blockchain/test/index.spec.ts

```diff
@@ -537,7 +538,8 @@ tape('blockchain test', (t) => {
       number: 15,
       parentHash: blocks[14].hash(),
       gasLimit: 8000000,
-      timestamp: blocks[14].header.timestamp.addn(1),
+      //eslint-disable-next-line
+      timestamp: toType(blocks[14].header.timestamp, TypeOutput.BigInt)! + BigInt(1),
```

Comment:
won't the timestamp be bigint already? another instance below as well L568
```suggestion
      timestamp: blocks[14].header.timestamp + BigInt(1),
```